trackman/__init__.py

```diff
@@ -172,6 +172,9 @@ def init_app():
         'CACHE_TYPE': "RedisCache",
         'CACHE_REDIS_URL': cache_redis_url,
     })
-
-
+#this command is sketchy, but it made running trackman locally work for me
```

Comment:
I think it would be best to have this triggered by an environment variable...perhaps DB_INIT_ON_BOOT or something like that.

Also, for PEP8 compliance, please add a space between `#` and the rest of the comment. It might also be good to add some newlines to clearly separate the "sketchy" bit out, but I don't know that that's strictly necessary.